README.md

```diff
@@ -16,24 +16,7 @@ It features:
 * Testing your project.
 * Benchmarking your project.
 
-#### How to install
```

Comment:
I don't think we should remove this content from the README, we still want the README to function as a quickstart guide without having to jump to other files.